cluster.ml

```diff
@@ -70,6 +70,7 @@ let deployment =
       ~zone:gcloud_zone
       ~java:`Oracle_7
       ~machine_type:(`Google_cloud `Highmem_8)
+      ~additional_packages:["postgresql-9.4"]
```

Comment:
Good idea, let me change this.

On Sat, Jul 23, 2016 at 1:09 PM Sebastien Mondet <notifications@github.com>
wrote:

> In cluster.ml
> <https://github.com/hammerlab/stratotemplate/pull/12#discussion_r71975094>
> :
>
> > @@ -70,6 +70,7 @@ let deployment =
> >        ~zone:gcloud_zone
> >        ~java:`Oracle_7
> >        ~machine_type:(`Google_cloud `Highmem_8)
> > +      ~additional_packages:["postgresql-9.4"]
>
> I can merge yes.
>
> I was thinking that the additional-packages should be in a
> configuration.env environment variable. That way other users can use the
> feature; and most users don't get a PG server running on every node for
> nothing.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hammerlab/stratotemplate/pull/12/files/9bdef675c6b46ff9bbe274a463cba95a32a80db5#r71975094>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKPe-19Fy-m5FGi_Ivo_3FVPC4FzNEZks5qYkqsgaJpZM4JTJEB>
> .
>